tests/recovery.py

```diff
@@ -10,7 +10,7 @@
 
 @reqs.description("Recovering a network")
 @reqs.recover(number_txs=2)
-def test(network, args, use_shares=False):
+def test(network, args, use_shares=True):
```

Comment:
We now test recovery with key shares in the e2e suite runs (instead of sealing key).